api/v1alpha1/openapi.yaml

```diff
@@ -3887,6 +3890,26 @@ components:
         - "DeviceUpdatedStatusOutOfDate"
         - "DeviceUpdatedStatusUpdating"
         - "DeviceUpdatedStatusUnknown"
+    DeviceLifecycleStatus:
+      type: object
+      required:
+        - status
+      properties:
+        status:
+          $ref: "#/components/schemas/DeviceLifecycleStatusType"
+        info:
+          type: string
+          description: "Human readable information about the device lifecycle status."
+      description: "Current status of the lifecycle of the device."
+    DeviceLifecycleStatusType:
+      type: string
+      enum:
```

Comment:
Nit: These types are missing the `x-enum-varnames` that are defined in other status types, eg `ApplicationsSummaryStatusType`.

As a result, the generated Typescript type has a different pattern that for other types:

```
export enum ApplicationsSummaryStatusType {
  ApplicationsSummaryStatusHealthy = 'Healthy',
  ApplicationsSummaryStatusDegraded = 'Degraded',
  ApplicationsSummaryStatusError = 'Error',
  ApplicationsSummaryStatusUnknown = 'Unknown',
}
```

vs

```
export enum DeviceLifecycleStatusType {
  ENROLLED = 'Enrolled',
  DECOMMISSIONING_REQUESTED = 'DecommissioningRequested',
  DECOMMISSIONING_STARTED = 'DecommissioningStarted',
  DECOMMISSION_ERROR = 'DecommissionError',
  DECOMMISSION_TIMEOUT = 'DecommissionTimeout',
  DECOMMISSIONED = 'Decommissioned',
}

```

It would be great to have the same pattern everywhere if possible.

Reply:
Thanks @celdrake , will be pushing a new version with this.